logging/tests/unit/handlers/test__helpers.py

```diff
@@ -63,6 +64,57 @@ def test_valid_context_header(self):
         self.assertEqual(trace_id, expected_trace_id)
 
 
+# webapp2 has not been ported to python3 yet, so do not try to test its
+# functionality there.
+if sys.version_info[0] == 2:
+    class Test_get_trace_id_from_webapp2(unittest.TestCase):
+
+        @staticmethod
+        def create_app():
```

Comment:
I was just following what the existing tests here do.  I have no strong feelings myself.  I do feel it's nice to keep all the logic for a test encapsulated in the test class, to make it clear what helper objects go with what test, but I'm happy to change it if you have a preference.